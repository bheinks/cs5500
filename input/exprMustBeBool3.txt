(* 
   Expression must be of type boolean
*)

program p;

begin
  if 't' then write('5');
  write('!')
end.

