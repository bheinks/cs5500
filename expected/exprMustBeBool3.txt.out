Line 8: Expression must be of type boolean
